packages/patternfly-4/react-core/.babelrc.js

```diff
@@ -1,15 +1,6 @@
-const babelENV = process.env.BABEL_ENV || 'development';
```

Comment:
Can we move the contents of this .babelrc.js file to packages/patternfly-4/ and then refer to it the same way everywhere instead of using the one in react-core in sibling folders?

Reply:
Done.